Marlin/src/lcd/tft/ui_320x240.cpp

```diff
@@ -682,12 +675,16 @@ void lcd_gotopixel(const uint16_t x, const lcd_uint_t y) {
   tft.set_background(COLOR_BACKGROUND);
 }
 
+void lcd_moveto(const lcd_uint_t col, const lcd_uint_t row) {
+  lcd_gotopixel(col * (TFT_COL_WIDTH), row * (MENU_ITEM_HEIGHT));
```

Comment:
i fear an overflow, i think lcd_uint_t is uint8_t for the 320x240 (320 > 255)
so >> (TFT_COL_WIDTH) * col  ?

Reply:
hmm seems ok in fact... c++ vs c i guess

Reply:
I may be mistaken, but I believe multiplication by a constant integer promotes to a signed integer before taking the final result.